duckdb-vortex/src/vortex_extension.cpp

```diff
@@ -19,28 +19,31 @@
 
 namespace duckdb {
 
+const uint32_t MAX_THREAD_COUNT = 192;
```

Comment:
Maybe add a comment why we picked that constant / that it is somewhat arbitrary.